example/test/exampleFluentDslWithTypeScriptAndPageObjects.spec.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Example for using the new Scenarioo Fluent DSL with TypeScript Typings of ScenariooJS.
+ *
+ * For TypeScript typings we only support Fluent DSL as our newest and most comfortable API
+ * that is recommended to be used.
+ *
+ * Precondition for running this example using labels in Fluent DSL:
+ * All labels used in Scenarioo Fluent DSL have to be defined before usage.
+ * Usually we do this once on starting up the test suite.
+ * The example `./exampleFluentDslLabelDefinitions.js` demonstrates the declaration of all labels
+ * used in this examples.
+ */
+import {browser, by, element} from 'protractor';
+import {scenario, step, useCase,} from '../../lib'; // use 'scenarioo-js' instead of '../../lib' in real project
+// label definitions: usually only declared once on setup (no need to import in every test):
+import './exampleFluentDslLabelDefinitions';
+import {StartPage} from './pages/startPage';
+
+/**
+ * The use case description
+ */
+useCase('Example Use Case with Fluent DSL in TypeScript')
+  .description('An optional but recommended description for the use case')
+  .labels(['example-custom-label'])
+  .describe(() => {
+    let startPage: StartPage;
+
+    beforeEach(() => {
+      startPage = new StartPage();
+    });
+    /**
+     * A scenario description
+     */
+    scenario('Example Scenario with Fluent DSL and reportStep Decorator')
+      .description('An optional but recommended description for the scenario')
+      .labels(['happy', 'example-label'])
+      .it(() => {
+        // the StartPage constructor navigates us to index.html
+
+        // A step on a decorator could also have additional propoerties, like e.g. labels
+        startPage.assertHeaderShown();
+
+        // do something on the page itself.
+        startPage.clickFirstListItem();
```

Comment:
```suggestion
        startPage.selectFirstListItem();
```
We should write test on use case or scenario level. If the goal is to select the element, we should name it accordingly. It does not matter how we do it (e.g. click).